src/chat/ContactSheet.tsx

```diff
@@ -16,14 +16,19 @@ import {ArrowRight, ExternalLink} from '@atb/assets/svg/mono-icons/navigation';
 import {Button} from '@atb/components/button';
 import {useAnalytics} from '@atb/analytics';
 import {useParkingViolationsReportingEnabled} from '@atb/parking-violations-reporting';
+import {Theme} from '@atb/theme/colors';
 
 type Props = {
   onReportParkingViolation: () => void;
 };
 
+const getInteractiveColor = (theme: Theme) => theme.color.interactive[2]
+const getBackgroundColor = (theme: Theme) => theme.color.background.neutral[0]
```

Comment:
Considering how many if these there are in this PR, we should probably look for a more ergonomic way to fetch specific contrast and interactive colors from the design system. Not sure how that will work now, when ids like "background_0" isn't a thing anymore. Do you have any thoughts?

But I'm all for merging this PR first, then finding another solution later 👍

Reply:
The nice thing here is that it is very explicit what variable we are fetching, but it would be nice having a `token` function, where we can just write `token('color.background.neutral.0')` and get the best of both worlds. Perhaps this could even align with the way we access these tokens in CSS (with a custom plugin we talked about last week).

Some ideas :) I will get this merged first!

Reply:
Yeah the explicitness of it is nice! We'll see if some nice solution presents itself, or if the current solution works as we get used to it 😄